README.md

```diff
@@ -1,29 +1,200 @@
 # Validations with `form_tag`
 
-## Objectives
+Now that we've learned to handle the server side of validations, we need to
+take care of the client side.
+
+At this point, we'll be in step three of the following flow:
+
+1. User fills out the form and hits "Submit", transmitting the form data via
+   a POST request.
+2. The controller sees that validations have failed, and re-renders the form.
+3. **The view displays the errors to the user**.
+
+# Objectives
+
+After this lesson, you'll be able to...
 
-- render or redirect based on validation of instance in create/update
 - Prefill in form values based on an instance
 - Print out full error messages based on an invalid instance
-- introspect on errors for a field
-- apply an error class to a field conditionally based on errors on a field
+- Introspect on errors for a field
+- Apply an error class to invalid fields
+
+
+# Pre-Filling Form Values
+
+No one likes re-doing work. First, let's make sure we know how to pre-fill
+forms with the user's input so they don't have to type everything all over
+again.
+
+In the next lesson, `form_for` will be handling this for us, but `form_for`
```

Comment:
I'd not mention `form_for` in such detail yet. Maybe something short about how there are two ways of handling forms in Rails, the form tag and `form_for`, and we'll be going over form tags first.